tests/system/presenter/test_get_user_scope.py

```diff
@@ -11,22 +11,22 @@ def test_good(self) -> None:
         self.set_models(
             {
                 "meeting/1": {
-                    "name": "meeting1",
                     "committee_id": 2,
                     "is_active_in_organization_id": 1,
                 },
+                # archived meeting
+                "meeting/2": {
+                    "committee_id": 2,
+                    "is_active_in_organization_id": None,
```

Comment:
To be correct `is_archived_in_organization_id` is necessary to determine an archived meeting